aspnetcore/security/data-protection/implementation/key-encryption-at-rest.md

```diff
@@ -3,100 +3,106 @@ title: Key encryption At rest in ASP.NET Core
 author: rick-anderson
 description: Learn implementation details of ASP.NET Core Data Protection key encryption at rest.
 ms.author: riande
-ms.date: 10/14/2016
+ms.date: 07/16/2018
 uid: security/data-protection/implementation/key-encryption-at-rest
 ---
 # Key encryption at rest in ASP.NET Core
```

Comment:
Key Vault is now available too.

Reply:
mmm ... These topics aren't playing well with each other. I've opened *Assess sufficiency of security topic content organization* #7666 to take a closer look.

For now, I'll patch the problem in this topic that surfaces protecting keys with Azure Key Vault.